caravel/assets/javascripts/SqlLab/components/DatabaseSelect.jsx

```diff
@@ -46,6 +43,12 @@ class DatabaseSelect extends React.Component {
 
 DatabaseSelect.propTypes = {
   onChange: React.PropTypes.func,
+  databaseId: React.PropTypes.number,
```

Comment:
i think this will probably through a warning when the default prop is set to `null` and it's expecting a number.

probably want to use `React.PropTypes.oneOfType`
````js
React.PropTypes.oneOfType([
  React.PropTypes.number,
  React.PropTypes.bool
]);
```

and make `databaseId: false` in the defaultProps
and databaseId